gm4_mending_tanks/data/mending_tanks/functions/init.mcfunction

```diff
@@ -0,0 +1,21 @@
+#announce module installation
+tellraw @a[gamemode=creative] ["",{"text":"[GM4]: Installing Mending Tanks..."}]
```

Comment:
Use translation keys for these tellraws. See the [template pack](https://github.com/Gamemode4Dev/GM4_Datapacks/blob/master/gm4_template_pack/data/template/functions/init.mcfunction) as an example. Also an issue on line 16

Reply:
I just copied liquid tanks I'll fix this.